pkg/controller/toolchainstatus/toolchainstatus_controller.go

```diff
@@ -406,10 +407,11 @@ func (r *ReconcileToolchainStatus) sendToolchainStatusNotification(logger logr.L
 		if err != nil {
 			return err
 		}
-		contentString = "<div><pre><code>" + string(statusYaml) + "</code></pre></div>"
+		contentString = "<div><pre><code>" + string(statusYaml) + "</code></pre></div>" // wrap with div/pre/code tags so the formatting remains intact in the delivered mail
 		subjectString = adminUnreadyNotificationSubject
 	case restoredStatus:
-		contentString = "<div><pre>ToolchainStatus is back to ready status.</pre></div>" // wrap with div/pre/code tags so the formatting remains intact in the delivered mail
+		environment = toolchainStatus.Status.RegistrationService.Health.Environment
```

Comment:
`toolchainStatus.Status.RegistrationService.Health.Environment` is useless here.
The name of this field is a bit misleading. It's actually not "environment" but rather "mode". So both our stage and prod is running in "prod" mode. Vs. "developer" or "test" modes.
What we actually need is the host cluster api server URL or the host cluster name...

Reply:
Also, if we add this environment thing then we need to add it to both notifications. Unready and restored.

Reply:
i believe member apiEndpoint would be fine as well, right?

Reply:
There can be multiple member clusters. We are interested in the host cluster here but not sure if it's easy to get it from this code...
I would leave it as is (without mentioning the env) and think about improving it later.